src/demokratie.md

```diff
@@ -110,8 +110,8 @@ dies zu belegen. Hier ist nur eine beispielhafte Liste:
 ## Transparenz
 
 Politik ist heute im Wesentlichen sehr intransparent. Die meisten Dokumente
-sind geheim. In den meisten Ländern gibt keine Verpflichtung diese zu
-veröffentlichen, und wenn doch, dann unterliegt dies strengen und schwer zu
+sind geheim. In den meisten Ländern gibt es keine Verpflichtung, diese zu
+veröffentlichen und wenn doch, dann unterliegt dies strengen und schwer zu
```

Comment:
Ich hatte mit dem Komma versucht die beiden Hauptsaetze hervorzuheben (das Komma is optional, aber korrekt). Allerdings ist der Satz ziemlich lang.. Vielleicht einfach einen Punkt machen?

```suggestion
sind geheim. In den meisten Ländern gibt es keine Verpflichtung, diese zu
veröffentlichen. Wenn doch, dann unterliegt dies strengen und schwer zu
```